src/routes/feed.xml.ts

```diff
@@ -0,0 +1,149 @@
+//Apple spec from here: https://help.apple.com/itc/podcasts_connect/#/itcb54353390
+//TODO implement all optional/situational tags as well
+const xml = (channel: Channel, episodes: Episode[]) => `
+<rss xmlns:atom="http://www.w3.org/2005/Atom" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd" xmlns:googleplay="http://www.google.com/schemas/play-podcasts/1.0" xmlns:media="http://www.rssboard.org/media-rss" version="2.0">
+<atom:link href="${channel.url}/feed.xml" rel="self" type="application/rss+xml"/>
+<channel>
+	<title>${channel.title}</title>
+	<itunes:title>${channel.title}</itunes:title>
+	<link>${channel.url}</link>
+	<language>en-us</language>
+	<copyright>&#169; 2022 ${channel.author}</copyright>
+	<itunes:author>${channel.author}</itunes:author>
+	<description>${channel.description}</description>
+	<itunes:type>episodic</itunes:type>
+	<itunes:owner>
+		<itunes:name>${channel.author}</itunes:name>
+		<itunes:email>${channel.email}</itunes:email>
+	</itunes:owner>
+	<itunes:image href="${channel.albumImageURL}"/>
+	<itunes:category text="${channel.category}"></itunes:category>
+	<itunes:explicit>${channel.explicit ? 'YES' : 'NO'}</itunes:explicit>
+
+${episodes
+	.map(
+		(episode) => `
+	<item>
+		<itunes:episode>${episode.number}</itunes:episode>
+		<itunes:episodeType>full</itunes:episodeType>
+		<itunes:season>${episode.season}</itunes:season>
+		<title>${episode.title}</title>		
+		<itunes:title>${episode.title}</itunes:title>
+		<description>
+			<![CDATA[
+			${episode.description}
+			]]>
+		</description>
+		<enclosure 
+			url="${episode.fileLocation}" 
+			length="${episode.length}" 
+			type="audio/mpeg"/>
+		<guid isPermaLink="false">${episode.guid}</guid>
+		<pubDate>${episode.published}</pubDate>
+		<itunes:duration>${episode.duration}</itunes:duration>		
+		<itunes:explicit>${episode.explicit ? 'YES' : 'NO'}</itunes:explicit>
+	</item>
+`,
+	)
+	.join('')}
+</channel>
+</rss>`;
+
+const processEpisodes = async (url: string): Promise<Episode[]> => {
+	const episodes = new Array<Episode>();
+	console.log(manifest);
+
+	await Promise.all(
+		Array.from({length: manifest.length}, async (_, i) => {
+			console.log(`${url}/${manifest[i]}`);
+			const file = await fetch(`${url}/${manifest[i]}`, {
+				method: 'GET',
+			});
+			const length = Number(file.headers.get('content-length'));
+			const type = file.headers.get('content-type')!;
+			const pubDate = file.headers.get('last-modified')!;
+			const meta = await mm.parseStream(file.body, {mimeType: type, size: length});
+
+			const episode = {
+				title: meta.common.title!,
+				description: meta.common.comment![0],
+				fileLocation: `${url}/${manifest[i]}`,
+				length,
+				duration: meta.format.duration!,
+				season: 1,
+				number: meta.common.track.no!,
+				published: new Date(pubDate),
+				explicit: false,
+				guid: meta.common.title!,
+			};
+			episodes.push(episode);
+		}),
```

Comment:
This was a bit more complicated than it needed to be, and also now catches extra unknown props on the `Episode` type.

```suggestion
	console.log(manifest);
	
	const episodes = await Promise.all(
		manifest.map(async (episodeFilename) => {
			console.log(`${url}/${episodeFilename}`);
			const file = await fetch(`${url}/${episodeFilename}`, {
				method: 'GET',
			});
			const length = Number(file.headers.get('content-length'));
			const type = file.headers.get('content-type')!;
			const pubDate = file.headers.get('last-modified')!;
			const meta = await mm.parseStream(file.body, {mimeType: type, size: length});

			const episode: Episode = {
				title: meta.common.title!,
				description: meta.common.comment![0],
				fileLocation: `${url}/${episodeFilename}`,
				length,
				duration: meta.format.duration!,
				season: 1,
				number: meta.common.track.no!,
				published: new Date(pubDate),
				explicit: false,
				guid: meta.common.title!,
			};
			return episode;
		}),
```